projects/cdk/utils/sorting/comparator.ts

```diff
@@ -0,0 +1 @@
+export type TuiComparator<T> = (a: T, b: T) => number;
```

Comment:
Do we need to move it here? Looks like it can remain in table

Reply:
fixed